tests/test_enhanced_census_utilities.py

```
@@ -322,19 +332,33 @@ def test_validate_download_url_success(self, mock_get):
-        assert is_valid is True
+            is_valid = self.discovery.validate_download_url("https://example.com/test.zip")
+            assert is_valid is True
+
+    @patch('requests.get')
+    def test_validate_download_url_ssl_refused_by_default(self, mock_get):
+        """Without SIEGE_INSECURE_SSL=1, an SSL failure is no longer
+        silently bypassed; the URL is reported as not valid."""
+        from requests.exceptions import SSLError
+        from siege_utilities.geo.boundary_result import BoundaryUrlValidationError
+        from siege_utilities.geo import spatial_data
+        with patch.object(spatial_data, "_ALLOW_INSECURE_SSL", False):
```

Ruff (0.15.12)

[warning] Boolean positional value in function call

(FBT003)